src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -853,15 +853,43 @@ class ApplicationManagerImpl
       const connection_handler::CloseSessionReason& close_reason) OVERRIDE;
 
 #ifdef ENABLE_SECURITY
-  // Overriden SecurityManagerListener method
+  /**
+   * @brief Notification about protection result
+   * @param connection_key Unique key of session which triggrs handshake
```

Comment:
@AKalinich-Luxoft typo

Reply:
@AByzhynar fixed in https://github.com/smartdevicelink/sdl_core/pull/1971/commits/a87c65251e81cb914deaa1f0fe1716175fe1270d